src/core_modules/capture-core/components/ListView/Filters/Contents/withButtons.js

```diff
@@ -89,17 +91,19 @@ export const withButtons = () => (InnerComponent: React.ComponentType<any>) =>
                                 muiButtonRef={this.setUpdateButtonInstance}
                                 primary
                                 onClick={this.handleUpdateClick}
+                                disabled={disabledUpdate}
                             >
-                                {i18n.t('Apply')}
+                                {i18n.t('Update')}
```

Comment:
Can you explain why this text label changes from `Apply` to `Update`?

Reply:
The text change is part of the new filters [redesign and functionality](https://docs.google.com/document/d/1BGxFZ6G64KcAS95Cmh0-ab5BzPbBGN5_AIh-2V5cVJA/edit#heading=h.pzj6mk7lxht4) . Thank you!